modules/core/core-schema/src/main/java/com/enonic/xp/core/impl/form/FormDefaultValuesProcessorImpl.java

```diff
@@ -32,10 +36,11 @@
     @Override
     public void setDefaultValues( final Form form, final PropertyTree data )
     {
-        processFormItems( form.getFormItems(), data, PropertyPath.from( "" ) );
+        processFormItems( form.getFormItems(), data, PropertyPath.from( "" ), null );
```

Comment:
maybe instead of null provide normalize it to "default" value `Occurrences.default()` or smth